src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.operation;
+
+import core.basesyntax.db.FruitStorage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void processOperation(FruitTransaction fruitTransaction) {
+        FruitStorage.getFruits().put(fruitTransaction.getFruit(),
+                fruitTransaction.getQuantity());
+    }

```

Comment:
This class should not begin with an empty line according to the checklist. Please remove the empty line at the beginning of the class definition.